### PR TITLE
Michal/common test/docs continuation

### PR DESCRIPTION
**Pull request by Mikaka27**
This polishes the documentation of common test further. Hopefully this is the last missing bit.

### CODE REVIEW

File: lib/common_test/src/ct_ftp.erl [hunk 1]
```diff
@@ -38,7 +36,11 @@ FTP client module (based on the `ftp` application).
 
 -define(DEFAULT_PORT,21).
 
--doc "For `target_name`, see module `m:ct`.".
+-doc """
+Reference to opened FTP connection associated to either a `handle` or `target_name`.
+
+For `target_name`, see module `m:ct`.
```
Comment (garazdawi) [inline lib/common_test/src/ct_ftp.erl:42]: I think we can delete this as the type itself contains a link now to where more information is available.

**Review by garazdawi — APPROVED** [review]

File: lib/common_test/src/ct_ftp.erl (hunk 1 above)
Comment (Mikaka27) [reply lib/common_test/src/ct_ftp.erl:42]: Will fix, thank you :)

Comment (Mikaka27) [reply lib/common_test/src/ct_ftp.erl]: Fixed this case, as well as other similar cases.

**Comment by garazdawi** [comment]
Building the CT docs generate a bunch of warnings: https://github.com/erlang/otp/actions/runs/9268808847/job/25498516518?pr=8516#step:4:2527

Can you fix those while at it?

```
#6 27.33      warning: documentation references function "ct_property_test:cmnd_names/1" but it is hidden
#6 27.33      │
#6 27.33  271 │   - `ct_property_test:cmnd_names/1` returns a list of commands (function calls)
#6 27.33      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 27.33      │
#6 27.33      └─ (common_test 1.27) ../src/ct_property_test.erl:271: :ct_property_test.present_result/5
#6 27.33 
#6 27.34      warning: documentation references function "ct_property_test:num_calls/1" but it is hidden
#6 27.34      │
#6 27.34  274 │   - `ct_property_test:num_calls/1` returns a list of the length of commands
#6 27.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 27.34      │
#6 27.34      └─ (common_test 1.27) ../src/ct_property_test.erl:274: :ct_property_test.present_result/5
#6 27.34 
#6 27.35      warning: documentation references function "ct_property_test:sequential_parallel/1" but it is hidden
#6 27.35      │
#6 27.35  276 │   - `ct_property_test:sequential_parallel/1` returns a list with information
#6 27.35      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 27.35      │
#6 27.35      └─ (common_test 1.27) ../src/ct_property_test.erl:276: :ct_property_test.present_result/5
#6 27.35 
#6 28.03 View "html" docs at "html/index.html"
#6 28.33      warning: documentation references function "ct_property_test:cmnd_names/1" but it is hidden
#6 28.33      │
#6 28.33  271 │   - `ct_property_test:cmnd_names/1` returns a list of commands (function calls)
#6 28.33      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 28.33      │
#6 28.33      └─ (common_test 1.27) ../src/ct_property_test.erl:271: :ct_property_test.present_result/5
#6 28.33 
#6 28.34      warning: documentation references function "ct_property_test:num_calls/1" but it is hidden
#6 28.34      │
#6 28.34  274 │   - `ct_property_test:num_calls/1` returns a list of the length of commands
#6 28.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 28.34      │
#6 28.34      └─ (common_test 1.27) ../src/ct_property_test.erl:274: :ct_property_test.present_result/5
#6 28.34 
#6 28.34      warning: documentation references function "ct_property_test:sequential_parallel/1" but it is hidden
#6 28.34      │
#6 28.34  276 │   - `ct_property_test:sequential_parallel/1` returns a list with information
#6 28.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#6 28.34      │
#6 28.34      └─ (common_test 1.27) ../src/ct_property_test.erl:276: :ct_property_test.present_result/5
#6 28.34 
```

**Comment by Mikaka27** [comment]
> Building the CT docs generate a bunch of warnings: https://github.com/erlang/otp/actions/runs/9268808847/job/25498516518?pr=8516#step:4:2527
> 
> Can you fix those while at it?
> 
> ```
> #6 27.33      warning: documentation references function "ct_property_test:cmnd_names/1" but it is hidden
> #6 27.33      │
> #6 27.33  271 │   - `ct_property_test:cmnd_names/1` returns a list of commands (function calls)
> #6 27.33      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 27.33      │
> #6 27.33      └─ (common_test 1.27) ../src/ct_property_test.erl:271: :ct_property_test.present_result/5
> #6 27.33 
> #6 27.34      warning: documentation references function "ct_property_test:num_calls/1" but it is hidden
> #6 27.34      │
> #6 27.34  274 │   - `ct_property_test:num_calls/1` returns a list of the length of commands
> #6 27.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 27.34      │
> #6 27.34      └─ (common_test 1.27) ../src/ct_property_test.erl:274: :ct_property_test.present_result/5
> #6 27.34 
> #6 27.35      warning: documentation references function "ct_property_test:sequential_parallel/1" but it is hidden
> #6 27.35      │
> #6 27.35  276 │   - `ct_property_test:sequential_parallel/1` returns a list with information
> #6 27.35      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 27.35      │
> #6 27.35      └─ (common_test 1.27) ../src/ct_property_test.erl:276: :ct_property_test.present_result/5
> #6 27.35 
> #6 28.03 View "html" docs at "html/index.html"
> #6 28.33      warning: documentation references function "ct_property_test:cmnd_names/1" but it is hidden
> #6 28.33      │
> #6 28.33  271 │   - `ct_property_test:cmnd_names/1` returns a list of commands (function calls)
> #6 28.33      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 28.33      │
> #6 28.33      └─ (common_test 1.27) ../src/ct_property_test.erl:271: :ct_property_test.present_result/5
> #6 28.33 
> #6 28.34      warning: documentation references function "ct_property_test:num_calls/1" but it is hidden
> #6 28.34      │
> #6 28.34  274 │   - `ct_property_test:num_calls/1` returns a list of the length of commands
> #6 28.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 28.34      │
> #6 28.34      └─ (common_test 1.27) ../src/ct_property_test.erl:274: :ct_property_test.present_result/5
> #6 28.34 
> #6 28.34      warning: documentation references function "ct_property_test:sequential_parallel/1" but it is hidden
> #6 28.34      │
> #6 28.34  276 │   - `ct_property_test:sequential_parallel/1` returns a list with information
> #6 28.34      │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> #6 28.34      │
> #6 28.34      └─ (common_test 1.27) ../src/ct_property_test.erl:276: :ct_property_test.present_result/5
> #6 28.34 
> ```

Sure, will take a look as well.

**Review by u3s — APPROVED** [review]

**Pull request merged**